term_open_posix.go

```diff
@@ -22,13 +22,21 @@ func Open(name string, options ...func(*Term) error) (*Term, error) {
 
 	orig, err := termios.Tcgetattr(uintptr(fd))
 	if err != nil {
+		_ = unix.Close(fd)
```

Comment:
no need to assign the value to _, just write `unix.Close(fd)`

Reply:
Fixed on both files and squashed.